README.md

```diff
@@ -165,6 +165,28 @@ File an issue or open a PR if you wish to contribute correct flags for your syst
 The introduction of new plugin functions in Mosquitto may result in some issue compiling versions 1.5.x and later.
 Please reach me with any solutions you may find when resolving said issues.
 
+#### Building the plugin on Raspberry Pi
+Make sure to edit the `Makefile` in `mosquitto-go-auth` before building.
+`Makefile` should look like this:
+```
+CFLAGS := -I/usr/local/include -fPIC
+LDFLAGS := -shared -Wl,-unresolved-symbols=ignore-all
```

Comment:
I think it'd be better to do the change directly in the `Makefile` as a special case, e.g.:
```
rpi:
	@echo "Bulding for $(UNAME_S)"
	env CGO_CFLAGS="$(CFLAGS)" go build -buildmode=c-archive go-auth.go
	env CGO_LDFLAGS="-shared -Wl,-unresolved-symbols=ignore-all" go build -buildmode=c-shared -o go-auth.so
	go build pw-gen/pw.go
```
Then we can document in README that for RPi you should run `make rpi` instead of simply `make`. What do you think?

Reply:
Good idea.